examples/playground/src/App.tsx

```diff
@@ -167,6 +169,7 @@ function App() {
         defaultUiSchema={uiSchema}
         defaultWidgets={widgets}
         isEditor={isEditor}
+        locale={locale}
```

Comment:
I think we should not pass locale to `VernaProvider` instead I'll wrap `VernaProvider` within `IntlProvider` then use `useIntl` in children components to get `locale`.

Reply:
Ohhh good idea, but how to we modify the locale if so? it should then be managed in a state on the client side right?